action/builder.go

```diff
@@ -137,5 +129,9 @@ func (b *EnvelopeBuilder) Build() Envelope {
 	if b.elp.version == 0 {
 		b.elp.version = version.ProtocolVersion
 	}
+	if b.elp.payload == nil {
+		panic("cannot build Envelope w/o a valid payload")
```

Comment:
why is panic added here?

Reply:
added to prevent such case -- `elp.payload` is nil --> no valid action is set --> meaningless to `build()`